### PR TITLE
I have Uploaded My Project....And ready for Review

### PR DESCRIPTION
**Pull request by VishnuKumarVerma**
Your given task is ready for review with all the necessary changes that you had told me to do i.e :

1. logo change
2. Make Hover Effects
3. Opening of calendar while entering our DOB
4. abbreviation on every essential elements

### CODE REVIEW

**Comment by Yash080902** [comment]
Okay vishnu
I will do that today
Tell this  also to Vishesh.


Well-done guys

**Comment by onlyVishesh** [comment]
sir i also done mostly all the things please check my pr

**Comment by Yash080902** [comment]
Okay vishesh I will do that
Sorry for the inconvenience because I am at my home.
Today we will confirm

File: Profile Page AMBU/image/edit.png
Comment (Yash080902) [inline Profile Page AMBU/image/edit.png:1]: Vishnu please make it more interactive okay 
even it is looking good
nice

File: Profile Page AMBU/image/notification.png
Comment (Yash080902) [inline Profile Page AMBU/image/notification.png:1]: try to change the color of the bell icon

File: Profile Page AMBU/index.html [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+                <div class="fifth-row-img">
+                  <a href="#">
+                    <abbr title="Edit"><img src="image/edit.png" /></abbr>
+                  </a>
+                </div>
+              </div>
+
+              <div class="sixth-row">
+                <div class="sixth-row-input">
+                  <label for="User-email">EMAIl ID</label>
```
Comment (Yash080902) [inline Profile Page AMBU/index.html:180]: Make the text of EMAIL proper either in block or small

**Review by Yash080902 — CHANGES_REQUESTED** [review]
Do these kind of changes ASAP

**Pull request closed without merging**